docker-compose.yaml

```diff
@@ -17,7 +17,8 @@ services:
              python3 manage.py tailwind build &&
              python3 manage.py collectstatic --noinput &&
              python3 manage.py runserver 0.0.0.0:9000"
```

Comment:
The prod server is using `manage.py runserver`? [Django docs](https://docs.djangoproject.com/en/4.2/ref/django-admin/) say:

> DO NOT USE THIS SERVER IN A PRODUCTION SETTING. It has not gone through security audits or performance tests. (And that’s how it’s gonna stay. We’re in the business of making web frameworks, not web servers, so improving this server to be able to handle a production environment is outside the scope of Django.)

Relates to my queries elsewhere of having ability to configure dev-only and prod-only stuff docker-compose.yml.

Reply:
Unfortunately so. We have uwsgi as part of our requirements and it's configured but for some reason it's not used. This will be rectified with the ansible setup.

Reply:
Ok, scratch that. In the deploy script there is a docker-compose.override.yaml which is different from the one committed in the repo, and that one changes runserver with uwsgi.